core/indigo-core/molecule/src/molecule_cdxml_saver.cpp

```diff
@@ -213,13 +214,13 @@ void MoleculeCdxmlSaver::addDefaultFontTable()
 
     name.readString("font", true);
     id = 3;
```

Comment:
it is hard to understand the logic with fonts. Please add constant for ids

Reply:
fixed